### PR TITLE
Added new sets and rebased

### PR DESCRIPTION
**Pull request by StatsGary**
Sorted out issues and rebased. @tomjemmett can you kindly have a look at next week and get these on.

### CODE REVIEW

File: vignettes/stranded_model.Rmd [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+## Feature engineering
+
+The next step will be to decide which features need to be engineered for our machine learning model. We will drop the admit_date and recode the frailty index, and perhaps allocate the age into age bands. 
+
+```{r feature_engineering}
+
+stranded_data <- stranded_data %>% 
+  mutate(stranded.label=factor(stranded.label)) %>% 
+  select(everything(), -c(admit_date))
```
Comment (tomjemmett) [inline vignettes/stranded_model.Rmd:58]: this is failing - I think there is a conflict on select with dplyr::select being masked. Need to fully qualify all calls to select throughout this Rmd

File: vignettes/stranded_model.Rmd [hunk 2]
```diff
@@ -0,0 +1,139 @@
+---
+title: "Stranded Model - a simulated dataset for machine learning classification"
+author: "Gary Hutson"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{LOS_model: A simulated hospital length-of-stay dataset}
```
Comment (tomjemmett) [inline vignettes/stranded_model.Rmd:7]: This title needs to be updated - causing vignette builds to gail

File: R/synthetic_news_data.R [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+#' @usage data(synthetic_news_data)
+#'
+#' @examples
+#' library(magrittr)
+#' library(dplyr)
+#' data("synthetic_news_data")
+#' synthetic_news_data %>% 
+#'  glimpse()
+#'
+
```
Comment (tomjemmett) [inline R/synthetic_news_data.R:37]: this space needs to be removed

File: R/synthetic_news_data.R [hunk 4]
```diff
@@ -0,0 +1,38 @@
+#' Synthetic National Early Warning Scores Data
+#'
+#' Synthetic NEWS data(\href{https://www.rcplondon.ac.uk/projects/outputs/national-early-warning-score-news-2}) as the results of the NHSR_synpop package.
```
Comment (tomjemmett) [inline R/synthetic_news_data.R:3]: \href{} is not necessary as vignettes are rendered with markdown - this will remove formatting warning in Rcmd check

**Review by tomjemmett — CHANGES_REQUESTED** [review]
the requestsed changes should solve issues with R cmd check. Make sure to run devtools::document to fix documentation

**Comment by StatsGary** [comment]
Thanks Tom. Will look and fix tomorrow.

**Comment by tomjemmett** [comment]
Cheers, sorry for the pedantry, but cran will moan otherwise! 🙂

**Comment by StatsGary** [comment]
No mate. I know how CRAN can be pal. Thanks bud ☺️

**Comment by StatsGary** [comment]
@tomjemmett checked that all checks have passed on Travis CI and is looking good. The issue was two packages needed to be bought in, so added to DESCRIPTION file.

**Review by tomjemmett — APPROVED** [review]

**Comment by chrismainey** [comment]
Good form @StatsGary .  Will try and get submitted to CRAN asap.

**Comment by StatsGary** [comment]
Cheers team CRAN 😁

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Pull request merged**